recipes/libheif/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "libheif" %}
+{% set version = "1.14.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/strukturag/libheif/releases/download/v{{ version }}/libheif-{{ version }}.tar.gz
+  sha256: d016905e247d6952cd7ee4f9b90957350b6a6caa466bc76fdfe6eb302b6d088c
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('libheif', max_pin='x.x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+  host:
+    - libde265
+    - libavif
+    - dav1d
+    - aom
+    - x265
+  run:
+    - libde265
+    - libavif
+    - dav1d
+    - aom
+    - x265
```

Comment:
```suggestion
```

All of the packages above have run_exports. These runtime dependencies will be automatically added.